enterprise/server/backends/pebble_cache/pebble_cache_test.go

```diff
@@ -2914,7 +2912,10 @@ func BenchmarkFindMissing(b *testing.B) {
 	}
 }
 
-func BenchmarkContains1(b *testing.B) {
```

Comment:
I think we should keep a separate "contains" benchmark for 2 reasons:
 - It's generally good to test the entirety of the public API. Contains happens to call FindMissing today but that doesn't have to generally be the case.
 -  Contains is a special case of FindMissing with a single digest which is not covered by the FindMissing benchmark which always does checks using batches.

Reply:
Added back.